features/step_definitions/openstudios_subdomain.rb

```diff
@@ -0,0 +1,6 @@
+Given(/^I'm on the subdomain (.*)$/) do |site_domain|
+  site_domain = 'openstudios.lvh.me' if site_domain == 'openstudios'
+
+  host! site_domain
+  Capybara.app_host = "http://#{site_domain}"
```

Comment:
does this get reset with each test/scenario?  I just wonder if it needs an `Around`

For the future, depending on how this goes, we might think about a decorator.  I guess it'll get weird if you jump between domains within a test.

Reply:
Let's chat about this. It's not being used right now because I couldn't figure out how to handle life cycle functionality with Cucumber.

Reply:
:+1:  i think something like in here https://relishapp.com/cucumber/cucumber/docs/writing-support-code/around-hooks where it does `Around("@one")` - we might be able to do something like with the spec support for elastic search.   `Around('@subdomain')` or `Around(@os_catalog)`